Astral Pathfinder/Models/Text.cpp

```diff
@@ -0,0 +1,56 @@
+//
+//  Text.cpp
+//  Astral Pathfinder
+//
+//  Created by Ian Holdeman on 2/24/18.
+//  Copyright © 2018 Logan Harvell, Ian Holdeman. All rights reserved.
+//
+
+
+// MARKL: Header File
+#include "Text.hpp"
+
+// MARK: Source Files
+#include "Game.hpp"
+#include "TextureManager.hpp"
+
+
+void Text::initDeposits() {
+  font = TTF_OpenFont("Resources/Assets/Arial.ttf", 24);
+  
+  rect.x = 1275;
+  rect.y = 600;
+  rect.w = 225;
+  rect.h = 50;
+}
+
+void Text::initFertility() {
+  font = TTF_OpenFont("Resources/Assets/Arial.ttf", 24);
+  
+  rect.x = 1275;
+  rect.y = 650;
+  rect.w = 225;
+  rect.h = 50;
+}
+
+void Text::update() {
+  
```

Comment:
In PlanetInfo, this could take in a planet and update all relevant textBoxes and controls